<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QDialog" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="drawing_controls_box">
     <property name="title">
      <string>Drawing settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="time_points_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="time_points_label">
          <property name="text">
           <string>Time between points:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="time_points_value">
          <property name="suffix">
           <string> second</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000100000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>0.005000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="time_contours_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="time_contours_label">
          <property name="text">
           <string>Time between contours:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="time_contours_value">
          <property name="suffix">
           <string> second</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000100000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>0.300000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="x_offset_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="x_offset_label">
          <property name="text">
           <string>Horizontal offset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="x_offset_value">
          <property name="suffix">
           <string> pixels</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="minimum">
           <number>-50</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="y_offset_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="y_offset_label">
          <property name="text">
           <string>Vertical offset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="y_offset_value">
          <property name="suffix">
           <string> pixels</string>
          </property>
          <property name="minimum">
           <number>-50</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="max_width_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="max_width_label">
          <property name="text">
           <string>Maximum table width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="max_width_value">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>150</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="max_height_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="max_height_label">
          <property name="text">
           <string>Maximum table height:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="max_height_value">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>70</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="image_proc_box">
     <property name="title">
      <string>Image processing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="cam_width_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="cam_width_label">
          <property name="text">
           <string>Camera width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="cam_width_value">
          <property name="suffix">
           <string> pixels</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>320</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="cam_height_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="cam_height_label">
          <property name="text">
           <string>Camera height:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="cam_height_value">
          <property name="suffix">
           <string> pixels</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>240</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="motors_controls_box">
     <property name="title">
      <string>Motors control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="up_table_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="up_table_label">
          <property name="text">
           <string>Table up angle:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="up_angle_value">
          <property name="suffix">
           <string> deg.</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>130</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="down_table_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="down_table_label">
          <property name="text">
           <string>Table down angle:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="down_angle_value">
          <property name="suffix">
           <string> deg.</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="home_angle_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="home_angle_label">
          <property name="text">
           <string>Arms home angle:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="home_angle_value">
          <property name="suffix">
           <string> deg.</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="resolution_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="resolution_label">
          <property name="text">
           <string>Microstep resolution:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="resolution_value">
          <property name="prefix">
           <string>1 / </string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>32</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="time_steps_Layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="time_steps_label">
          <property name="text">
           <string>Time between steps:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="time_steps_value">
          <property name="suffix">
           <string> second(s)</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000100000000000</double>
          </property>
          <property name="maximum">
           <double>2.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>0.005000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_Layout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dialog_buttons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>accepted()</signal>
   <receiver>Configuration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>rejected()</signal>
   <receiver>Configuration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
